tasks.py

```diff
@@ -678,6 +677,73 @@ def get_day_of_week_code(day):
     __schedule_json_to_dynamo(ctx, schedule_data=schedule_data)
 
 
+@task(pre=[production_failsafe])
+def import_fas_schedule_from_csv(ctx, filepath):
+    columns = (
+        "course_code",
+        "title",
+        "day",
+        "start",
+        "end",
+        "zoom_link",
+        "opencast_series_link",
+        "crn",
+        "producer",
+        "sit_in"
+    )
+    day_of_week_map = {
+        "Monday": "M",
+        "Tuesday": "T",
+        "Wednesday": "W",
+        "Thursday": "R",
+        "Friday": "F"
+    }
+    with open(filepath, "r") as f:
+        reader = csv.reader(f)
+        next(reader)
+        rows = [ dict(zip(columns, row)) for row in reader]
+
+    schedule_data = {}
+    for row in rows:
+
+        zoom_link = urlparse(row["zoom_link"])
+        if not zoom_link.scheme.startswith("https"):
+            print("Invalid zoom link value for {}: {}" \
+                  .format(row["course_code"], row["zoom_link"]))
+            continue
+        zoom_series_id = urlparse(row["zoom_link"]).path.split("/")[-1]
+        schedule_data.setdefault(zoom_series_id, {})
+        schedule_data[zoom_series_id]["zoom_series_id"] = zoom_series_id
+
+        opencast_series_id = urlparse(row["opencast_series_link"]) \
+            .fragment.replace("/", "")
+        schedule_data[zoom_series_id]["opencast_series_id"] = opencast_series_id
+
+        topic = "{} - {}".format(row["course_code"], row["title"])
+        schedule_data[zoom_series_id]["opencast_topic"] = topic
```

Comment:
So, actually, this was "opencast_subject" in all my code.
We can change it to "opencast_topic" if you want.
Then I will update the spring schedule as well. It's confusing to try to read through the dynamo table with two columns for the same data.

Reply:
man i coulda sworn i saw "_topic" somewhere. no, i'll change it to subject. i'll do a double update of the schedule (both sources) once this is merged and hopefully it'll get rid of all the items with the wrong column.